### PR TITLE
add additional os information

### PR DESCRIPTION
**Pull request by mikethebeer**

### CODE REVIEW

**Comment by chaudum** [comment]
jvm type missing?

File: docs/sql/system.txt [hunk 1]
```diff
@@ -450,6 +450,25 @@ os_info
...
 +-------------------------------------+----------------------------------------------+-------------+
+| ``os_info['name']``                 | Name of the operating system (ex: Linux,     | ``String``  |
+|                                     | Windows, Mac OS X)                           |             |
++-------------------------------------+----------------------------------------------+-------------+
+| ``os_info['arch']``                 | Name of the JVM architecture (ex: amd64,     | ``String``  |
+|                                     | x86)                                         |             |
++-------------------------------------+----------------------------------------------+-------------+
+| ``os_info['version']``              | Version of the operating system              | ``String``  |
++-------------------------------------+----------------------------------------------+-------------+
+| ``os_info['jvm']``                  | Information about the JVM                    | ``Object``  |
```
Comment (chaudum) [inline docs/sql/system.txt]: JVM (Java Virtual Machine)

File: docs/sql/system.txt [hunk 2]
```diff
@@ -450,6 +450,25 @@ os_info
...
++-------------------------------------+----------------------------------------------+-------------+
+| ``os_info['jvm']['version']``       | The JVM version                              | ``String``  |
++-------------------------------------+----------------------------------------------+-------------+
+| ``os_info['jvm']['vm_name']``       | The name of the JVM (eg. OpenJDK,            | ``String``  |
+|                                     | Java Hotspot(TM) )                           |             |
++-------------------------------------+----------------------------------------------+-------------+
+| ``os_info['jvm']['vm_vendor']``     | The JVM vendor name                          | ``String``  |
```
Comment (chaudum) [inline docs/sql/system.txt]: The vendor name of the VM.

**Comment by chaudum** [comment]
I am not sure whether it is good to end all descriptions with `.` or not. However we should be consistent.

**Comment by mikethebeer** [comment]
@chaudum since its not a whole sentence there shouldn't be a dot (.) at the end.

**Comment by chaudum** [comment]
ack

**Pull request merged**